nmigen/lib/fifo.py

```diff
@@ -506,10 +506,15 @@ def elaborate(self, platform):
             fifo.w_data.eq(self.w_data),
             self.w_rdy.eq(fifo.w_rdy),
             fifo.w_en.eq(self.w_en),
-            self.w_level.eq(fifo.w_level),
         ]
 
-        m.d[self._r_domain] += self.r_level.eq(fifo.r_level + self.r_rdy - self.r_en)
+        buffer_used = Signal()
```

Comment:
I think this code would be easier to understand if the signals subject to CDC used the same naming conventions as the ones in `AsyncFIFO`; something like `r_consume_buffered` and `w_consume_buffered`.